drivers/scheduler/k8s/k8s.go

```diff
@@ -126,6 +126,7 @@ const (
 	k8sNodeReadyTimeout    = 5 * time.Minute
 	volDirCleanupTimeout   = 5 * time.Minute
 	k8sObjectCreateTimeout = 2 * time.Minute
+	k8sObjectCloneSnapshotTimeout = 4 * time.Minute
```

Comment:
Amol - Since this timeout is applied for pod creation, should we call it as k8sPodCreateTimeout?

Reply:
Done